Mathlib/Analysis/NormedSpace/Ray.lean

```diff
@@ -43,7 +43,7 @@ theorem norm_sub (h : SameRay ℝ x y) : ‖x - y‖ = |‖x‖ - ‖y‖| := by
   wlog hab : b ≤ a with H
   · rw [SameRay.sameRay_comm] at h
     rw [norm_sub_rev, abs_sub_comm]
-    have := @H E _ _ F
+    have := @H E _ _ ℝ
```

Comment:
This introduced a useless `F` in the statement (that is not present in the mathlib3 version).

Reply:
Sorry, I'm not sure how that snuck in!